src/rhelocator/update_images/gcp.py

```diff
@@ -57,3 +57,65 @@ def normalize_google_images(image_list: list[Any]) -> list[dict[str, str]]:
         normalized_images.append(image_data)
 
     return normalized_images
+
+
+def parse_image_version_from_name(image_name: str) -> str:
+    """Parse an google image name and return version string.
+
+    Args:
+        image_name: String containing the image name, such as:
+                    rhel-7-9-sap-v20220719
+
+    Returns:
+        Returns the google image version as string
+                    rhel-7-9
+    """
+    res = image_name.split("-", 3)
+    return res[0] + "-" + res[1] + "-" + res[2]
```

Comment:
Hmm I'm afraid Google images don't always follow the RHEL-MAJOR-MINOR-vVERSION formula. 
Sometimes the minor is missing: rhel-9-v20220719, rhel-9-arm64-v20220719 etc. If you want to we can wipe up a quick regex for that.

Reply:
Just as an example: https://regex101.com/r/CiABs5/1

Reply:
Oh yeah right - thank you! :)